common/lib/nft/kautomarket.ts

```diff
@@ -281,8 +280,8 @@ async function* setDefautCollectionRoyalty(
 const getReceiverAmount = (price: BigNumberish = 0, fee = 0): BigNumber => BigNumber.from(price).mul(fee).div(MAX_FEE);
 
 const isValidPrice = (price: BigNumberish = 0, minRoyaltyAmount: BigNumberish = 0): boolean =>
-  BigNumber.from(price).eq(0) ||
-  BigNumber.from(price).sub(getReceiverAmount(price, config.treasury.fee)).sub(minRoyaltyAmount).gt(0);
+  BigNumber.from(price).gt(getReceiverAmount(price, config.treasury.fee).add(minRoyaltyAmount)) ||
```

Comment:
this was to permit to set nft price to "0".
Because if "0" is not a valid price, the "Remove from sale" button doesn't work
Do you think we have to find another solution to make this button working ?